packages/kbn-storybook/lib/default_config.ts

```diff
@@ -6,8 +6,10 @@
  * Side Public License, v 1.
  */
 
+import * as path from 'path';
 import { StorybookConfig } from '@storybook/core/types';
 
+const toPath = (_path: string) => path.join(process.cwd(), _path);
```

Comment:
Can we use REPO_ROOT from `./lib/constants` to be safe here?  I don't know of a workflow that breaks..but relative scripts calls may be error.